README.md

```diff
@@ -50,7 +50,7 @@ The plugin includes a [Recommendations Block](https://docs.parse.ly/recommendati
 #### Advanced integrations support
 
 While the plugin works out of the box for basic integrations, it offers a host of features that easily allow for advanced integration scenarios:
-- Support for [Google Tag Manager, AMP, Facebook Instant Articles, Google Web Stories and Cloudflare](https://docs.parse.ly/plugin-common-questions/#h-is-wp-parsely-compatible-with-amp-facebook-instant-articles-or-google-web-stories) is included.
+- Support for [Google Tag Manager, AMP, Google Web Stories and Cloudflare](https://docs.parse.ly/plugin-common-questions/#h-is-wp-parsely-compatible-with-amp-facebook-instant-articles-or-google-web-stories) is included.
```

Comment:
Should we also update the info on this page or maybe also remove `facebook` from the link?

Reply:
Yes, as far as I know, the docs will be updated by @thompsonjoshua. @thompsonjoshua, if you change the page's link please let us know so we can update our readme.